addons/stock/report/stock_report_views.xml

```diff
@@ -113,13 +113,20 @@
             file="stock.label_lot_template_view"
             report_type="qweb-text"
         />
-        <report id="label_transfer_template"
+        <report id="action_label_transfer_template_zpl"
```

Comment:
maybe rename the view accordingly 
stock.label_transfer_template_view -> stock.label_transfer_template_view_zpl

Reply:
@sle-odoo  done

Reply:
👍